### PR TITLE
[FIX] base: fix access right mismatch warning

### PR DESCRIPTION
**Pull request by niyasraphy**
before this commit, if group has no category assigned for it, in the user group information message it shows the group name in the format False: Group name, where false is the category name.

* open users form view
* assign administrator role in sales and editor role in website
* Access Rights Mismatch? Since Marc Demo is a/an "Website: Editor and Designer", they will at least obtain the right "False: Bypass HTML Field Sanitize"

after this commit, in the users form view, in the warning/information message, instead of False it will show Other as the the group is displayed in the Other section in users form view.

Screenshot from 2023-01-27 17-48-30

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/addons/base/models/res_users.py [hunk 1]
```diff
@@ -1629,12 +1629,12 @@ def _prepare_warning_for_group_inheritance(self, user):
             )
             if missing_implied_groups:
                 # prepare missing group message, by categories
-                missing_groups[group] = ", ".join(f'"{missing_group.category_id.name}: {missing_group.name}"'
+                missing_groups[group] = ", ".join(f'"{missing_group.category_id.name if missing_group.category_id else _("Other")}: {missing_group.name}"'
```
[the remark this answers is not in the document]
Comment (tde-banana-odoo) [reply odoo/addons/base/models/res_users.py:1632]: Uncategorized ?

[the remark this answers is not in the document]
Comment (niyasraphy) [reply odoo/addons/base/models/res_users.py]: @xmo-odoo updated the pr. 
currently when the category is not assigned for the user group, it is displayed under Other section in user form, thus i have given Other in the pr.

Screenshot from 2023-01-30 19-51-57

@tde-banana-odoo thanks for the suggestion, so which should we go with ?

thanks

[the remark this answers is not in the document]
Comment (tde-banana-odoo) [reply odoo/addons/base/models/res_users.py]: Other is ok for me, and don't forget to export it in translation :)

[the remark this answers is not in the document]
Comment (niyasraphy) [reply odoo/addons/base/models/res_users.py]: @tde-banana-odoo i tried updating the pot file and i see that the term Other is already part of the pot file from the same python file

**Comment by tde-banana-odoo** [comment]
@robodoo r+

Thanks for updating :) the group management / formatting could be refactored a bit indeed as pointed out by Xavier, but as a quick fix in stable I think it is ok :)

**Pull request closed without merging**